Vagrantfile

```diff
@@ -80,6 +80,13 @@ $provision_script = <<SCRIPT
 set -x
 set -e
 set -o pipefail
+# If the host is running SELinux remount the /sys/fs/selinux directory as read only,
+# needed for apt-get to work.
+if [ -d "/sys/fs/selinux" ]; then
+  sudo mount -o remount,ro /sys/fs/selinux
+fi
+# Set default locale, this prevents errors on vagrant ssh
+echo "LC_ALL=en_US.UTF-8" | sudo tee -a /etc/default/locale
```

Comment:
There is an [`update-locale`](http://manpages.ubuntu.com/manpages/xenial/man8/update-locale.8.html) command for this. Note, however, that “[Using LC_ALL is strongly discouraged as it overrides everything. Please use it only when testing and never set it in a startup file.](https://wiki.debian.org/Locale)” You should probably remove LC_ALL from your outside configuration instead of adding it inside.